deployment/ccip/changeset/state.go

```diff
@@ -24,7 +24,9 @@ import (
 	"github.com/smartcontractkit/chainlink-ccip/chains/evm/gobindings/generated/v1_5_0/evm_2_evm_offramp"
 	"github.com/smartcontractkit/chainlink-ccip/chains/evm/gobindings/generated/v1_5_0/evm_2_evm_onramp"
 
+	"github.com/smartcontractkit/chainlink-ccip/chains/evm/gobindings/generated/latest/factory_burn_mint_erc20"
```

Comment:
why latest? should we not use versioned folder following the convention?

Reply:
Unfortunately we don't have these in a versioned folder, I can make this change in chainlink-ccip

Reply:
https://github.com/smartcontractkit/chainlink-ccip/pull/836